vignettes/contribution_model.Rmd

```diff
@@ -15,22 +15,20 @@ knitr::opts_chunk$set(
 )
 ```
 
-#Introduction 
+# Introduction 
 
 This vignette gives you an overview of the anticipated contribution model.
 The goal is to enable users of admiral to contribute and test the contribution model for admiral. Adjustments might be done over time.
  
-Our contribution model is still in a testing phase so we advise you to contact us via [Slack](https://app.slack.com/client/T028PB489D3/C02C8UN2JTT) when an issue is created to discuss details or align expectations if you are not familiar with the admiral philosophy and programming strategy. The team will try to review the issues within the next sprint and give some initial feedback. Since we are not a 100% fully resourced software development team it might be that some issues will take longer to respond to depending on the amount of overall issues. 
+Our contribution model is still in a testing phase so we advise you to contact us via [Slack](https://pharmaverse.slack.com/archives/C02M8KN8269) when an issue is created to discuss details or align expectations if you are not familiar with the admiral philosophy and programming strategy. The team will try to review the issues within the next sprint and give some initial feedback. Since we are not a 100% fully resourced software development team it might be that some issues will take longer to respond to depending on the amount of overall issues. 
 
 
 # Basics
- * For each new contribution, the user or Data Scientist creates an issue on the issue tab on [Git](https://github.com/Roche-GSK/admiral/issues) and put it in our backlog.
+ * For each new contribution, the user or Data Scientist creates an issue on the issue tab on [GitHub](https://github.com/pharmaverse/admiral/issues) to put it in our backlog. 
  
- * Label each issue with a label named “user request” so we can easily identify new requests.
+ * Familiarize yourself with our [programming strategy](programming_strategy.html), guidance for [GitHub usage](git_usage.html) and unit testing and the overall [goal of admiral](index.html).
```

Comment:
I think it should be `(../index.html)` instead of `(index.html)`.

We could also link to the unit test vignette.